microsoft-365/admin/add-users/remove-former-employee-step-4.md

```diff
@@ -19,175 +19,46 @@ ms.custom:
 - TRN_M365B
 - OKR_SMB_Videos
 - AdminSurgePortfolio
-- AdminTemplateSet
 - m365solution-removeemployee
 search.appverid:
 - BCS160
 - MET150
 - MOE150
-description: "Follow the steps in this article to give another employee access to the former employee's OneDrive and Outlook data."
+description: "Follow these steps to forward a former employee's email to another employee or convert to a shared mailbox."
 ---
 
-# Step 4 - Give another employee access to OneDrive and Outlook data
+# Step 4 - Forward a former employee's email to another employee or convert to a shared mailbox
 
-When an employee leaves your organization, you'll want to access their OneDrive and Outlook data, back it up, and choose whether to give it to another employee.
-  
-## Access a former user's OneDrive documents
+In this step, you assign the former employee's email address to another employee, or convert the user's mailbox to a shared mailbox.
 
-If you remove a user's license but don't delete the account, you can give yourself access to the content in the user's OneDrive. If you delete the user's account, you have 30 days by default to access the former user's OneDrive data. [Learn how to set the OneDrive retention for deleted users](/onedrive/set-retention). If you don't [restore a user account](/office365/admin/add-users/restore-user) within this time, their OneDrive content is deleted.
+## Convert former employee's mailbox to a shared mailbox
 
-To preserve a former user's OneDrive files, first give yourself access to their OneDrive, and then move the files you want to keep.
+When you convert a user's mailbox to a shared mailbox, all of the existing email and calendar is retained. Only now it's in a shared mailbox where several people will be able to access it instead of one person. You can convert a shared mailbox back to a user (private) mailbox at a later date if you want.
 
-1. In the admin center, go to the **Users** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=834822" target="_blank">Active users</a> page.  
+- Creating a shared mailbox is the less expensive way to go because you won't have to pay for a license **as long as the mailbox is smaller than 50GB**. Over 50GB and you'll need to assign a license to it.
```

Comment:
```suggestion
- Creating a shared mailbox is the less expensive way to go because you won't have to pay for a license **as long as the mailbox is smaller than 50 GB**. If it is over 50 GB, you'll need to assign a license to it.
```